docs/user-guide/faq.md

```diff
@@ -29,6 +29,8 @@ The [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#managing-plugins-w
 
 When using NixVim it is possible to encounter an error of the type `attribute 'name' missing`, for example it could look like:
 
+<!-- TODO: Update example now that we use `mkPackageOption` -->
```

Comment:
Did we want to handle this in this PR or follow up?

Reply:
I think we'll leave it for later. Next time a user reports a mismatched nixpkgs error we can use the error message they report in the example here.